### ISSUE TITLE
bug: wss and audio not working with your proxy

### ISSUE THREAD

**Issue by ashuvax** [requestly/requestly#972]
### Describe your issue?

wss and audio not working with your proxy

### Repro steps

In wss, the request is closed immediately after the handshake,
In audio it waits a long time without receiving a response

### What Requestly tool were you using?

- [ ] Extension
- [X] Desktop-App
- [ ] Android SDK
- [ ] Selenium

### Your Environment

windows 11

### Requestly Version

Unknown, not written in the desktop app

**Comment by ashuvax** [comment]
> Hey @ashuvax. Thanks for reporting this issue. We've fixed the WSS issue in this PR [requestly/requestly-proxy#26](https://github.com/requestly/requestly-proxy/pull/26). We'll release a new version by tomorrow.
> 
> We are still looking into the audio issue. Will keep you updated on the status

The wss has not yet been addressed
You can try on WhatsApp Web

**Comment by nsrCodes** [comment]
Hey @ashuvax 
I tried using whatsapp web in chrome (both in a new and existing chrome profile) it seems to be working fine for me. Can you please explain what exactly didn't work for you?

And which version of requestly desktop app are you using?

**Comment by nsrCodes** [comment]
I am using the desktop app to setup the proxy to my browser, how are you applying the proxy @ashuvax ?

**Comment by ashuvax** [comment]
@nsrCodes I click here:
![image](https://github.com/requestly/requestly/assets/78299418/c79acdfe-6036-46b5-9e54-6da0436f9c88)

**Comment by nsrCodes** [comment]
can maybe you try by clicking on `Connect Apps` button. This will open a modal where you can choose which browser you want to connect to. This will only apply the proxy for the browser that you specify

**Comment by ashuvax** [comment]
Thanks for the idea, I'll check it out
Anyway, this is a bug that needs to be addressed

**Comment by ashuvax** [comment]
@nsrCodes You are right, now it works for me, I see that there is a problem only when I work with a provider that filters my network, I will contact him.
But I don't see the request in the requestly interface and I can't do actions like redirects and all the other things that can be done in a request

**Comment by nsrCodes** [comment]
you should be able to see the intercepted requests inside the network traffic tab. In case you can't see the requests, make sure that you do not have any filters set, like a search term. 
<img width="497" alt="Screenshot 2023-09-28 at 6 00 03 PM" src="https://github.com/requestly/requestly/assets/57226514/ff8047cb-5d93-4dcd-a7a3-03c9a79e79b0">

And to apply the rules, just right click on the request you want to modify. This will show you all the possible modification options.
<img width="762" alt="Screenshot 2023-09-28 at 6 01 54 PM" src="https://github.com/requestly/requestly/assets/57226514/58732856-224d-4e3c-b027-70917546f630">

**Comment by ashuvax** [comment]
I don't have filters set,
I don't see it with you either,
Can you show me that you do see the wss request?

**Comment by nsrCodes** [comment]
Hey @ashuvax 
You are right, the wss request is not shown in the network inspector inside requestly. Currently we don't have support to log and show web socket requests.

Thanks for pointing this out. I will add this to our feature backlog and discuss it further with the team.

**Comment by Ronakkadhi** [comment]
Closing as stale — no activity for over a year.

As part of the 2026 repository restructure, we're closing dormant issues to focus on actively-tracked items. If this is still relevant, please re-open or file a fresh issue at the right home:

- 🌐 **HTTP Interceptor** (Chrome extension + web app + desktop proxy) → [requestly/interceptor](https://github.com/requestly/interceptor/issues/new/choose)
- 🚀 **Requestly API Client** → this repo

Thanks for taking the time to file the original report.

**Comment by Ronakkadhi** [comment]
Reopening — reverting the bulk-close while we test the migration flow on a smaller sample first. Sorry for the inbox noise.
